### PR TITLE
fix: attempting rollup build

### PR DESCRIPTION
**Pull request by bashleigh**
# Pull request checklist

**Detail as per issue below (required):**

fixes: #

### CODE REVIEW

File: packages/login-with-reapit/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  connectOAuthUrl: 'https://connect.reapit.cloud', // Our OAuth url
+  connectLoginRedirectPath: '', // Your logout page path (when logging in)
+  connectLogoutRedirectPath: '/login', // Your login page path (when logging out)
+  rootElement: document.querySelector('#reapit-connect-component'), // Your Element on the page
+  connectHasSessionCallback, // The callback for when the session is created
+  companyName: "Test Company", // Your Company name
+})
+```
+
+## Shadow Root Element
```
Comment (joshbalfour) [inline packages/login-with-reapit/README.md:36]: did you mean to finish this or is that coming in another PR?

File: packages/login-with-reapit/src/components/logout.svelte [hunk 2]
```diff
@@ -23,5 +25,3 @@
     background-color: #23a4de;
   }
 </style>
-
-<button on:click={handleLogoutClick}><SignOutButton /></button>
```
Comment (joshbalfour) [inline packages/login-with-reapit/src/components/logout.svelte:27]: why the rearrange here? (just curious)

**Review by joshbalfour — APPROVED** [review]
LGTM, just a couple of questions (not blocking merging) 👍

File: packages/login-with-reapit/README.md (hunk 1 above)
Comment (bashleigh) [reply packages/login-with-reapit/README.md:36]: yes, I need to work this bit out. Now I've built the package I can deploy + test it out

File: packages/login-with-reapit/src/components/logout.svelte (hunk 2 above)
Comment (bashleigh) [reply packages/login-with-reapit/src/components/logout.svelte:27]: Will's change, makes sense as you want the element below the style tags

**Pull request merged**